internal/fuse/file.go

```diff
@@ -20,47 +21,31 @@ import (
 const blockSize = 512
 
 // Statically ensure that *file implements the given interface
```

Comment:
This comment is now stale because one of the checks is for *openFile. You could update it, but removing it may be easier.

Reply:
the comment is now updated.